src/main/proto/io/dartinc/registry/v1beta1/registry.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package io.dartinc.registry.v1beta1;
+
+option java_multiple_files = true;
+
+import "tech/figure/util/v1beta1/document.proto";
+import "tech/figure/util/v1beta1/types.proto";
+import "validate/validate.proto";
+
+/**
+Digital Asset Registry Technology (DART) ENote metadata
+ */
+message ENote {
```

Comment:
Will this ENote message be inside a Loan Package? otherwise, we'll need a loan ULI or ID here

Reply:
It will be inside a Loan Package and the scope ID will be the ULI converted to UUID, so I think we're covered either way.